chipy/videos/15261.json

```
@@ -0,0 +1,38 @@
+{
+  "category": "ChiPy",
+  "copyright_text": "CC BY",
+  "description": "A deep dive into what actually happens when you're interfacing with GPIO pins at the hardware and register level in MicroPython.",
+  "duration": 1260,
+  "language": "eng",
+  "quality_notes": null,
+  "recorded": "2023-05-11T19:40:00",
+  "slug": "Micropython_gpio",
+  "source_url": "https://youtu.be/6wc452U2Gzw",
+  "speakers": [
+    "Andrew Wingate"
+  ],
+  "summary": "",
+  "tags": [
+    "#MicroPython #Electronics"
```

Tags should be comma-separated to ensure that they're parsed individually by the site build process.

```suggestion
    "MicroPython", "Electronics"
```